src/components/open-media-button.js

```diff
@@ -1,18 +1,27 @@
-const hubsRegex = /https?:\/\/(hubs.local(:\d+)?|(smoke-)?hubs.mozilla.com)\/(\w+)\/?\S*/;
-const isHubsUrl = hubsRegex.test.bind(hubsRegex);
+import { isHubsSceneUrl, isHubsRoomUrl } from "../utils/media-utils";
 
 AFRAME.registerComponent("open-media-button", {
   init() {
     this.label = this.el.querySelector("[text]");
 
     this.updateSrc = () => {
       this.src = this.targetEl.components["media-loader"].data.src;
-      this.label.setAttribute("text", "value", isHubsUrl(this.src) ? "visit" : "open link");
+
+      let label = "open link";
+
+      if (isHubsSceneUrl(this.src)) {
+        label = "use scene";
+      } else if (isHubsRoomUrl(this.src)) {
+        label = "visit";
```

Comment:
We should maybe say "visit room" or "visit hub"

Reply:
yep good point, since right now it's not clear it's a room